v3/src/data-interactive/handlers/component-handler.ts

```diff
@@ -150,14 +150,30 @@ export const diComponentHandler: DIHandler = {
 
     if (!values) return valuesRequiredResult
 
-    const v2Type = getTileContentInfo(content.type)?.getV2Type?.(content) ?? kComponentTypeV3ToV2Map[content.type]
-    const handler = diComponentHandlers.get(v2Type)
-    if (handler) {
-      // TODO: better error message?
-      return handler.update?.(content, values) ?? errorResult(t("V3.DI.Error.notFound"))
-    }
+    let result: DIHandlerFnResult | undefined
+    component.applyModelChange(() => {
+      // Handle updating generic component features
+      const { cannotClose, dimensions, position, title } = values as V2Component
+      if (cannotClose != null) component.setCannotClose(cannotClose)
+      if (title) component.setTitle(title)
+      // TODO Handle string positions?
+      const _position = position && typeof position !== "string" ? position : undefined
```

Comment:
How about:
```suggestion
      const _position = typeof position === "string" ? +position : position
```
which will attempt to convert the string to a number, if possible.

Reply:
The strings I was thinking of here are "top" and "bottom", which are legal options for `position` when creating components (though not yet implemented in v3). However, it seems like v2 ignores these `position` options for `update component`, so I think it should be alright to just ignore all strings here.